src/bindings/RescriptCompilerApi.res

```diff
@@ -20,13 +20,12 @@ module Lang = {
     | OCaml => "ml"
     }
 
-  let decode = (json): t => {
-    open! Json.Decode
-    switch string(json) {
+  let ofString = lang => {
+    switch lang {
     | "ml" => OCaml
     | "re" => Reason
     | "res" => Res
-    | other => raise(DecodeError(`Unknown language "${other}"`))
```

Comment:
Why drop this and not create our own Error with the same message? assert(false) is not very actionable unfortunately.

Reply:
Fixed